scene_select/utils.py

```diff
@@ -119,22 +118,23 @@ def scene_move(current_path: Path, current_base_path: str, new_base_path: str):
         with Datacube(app="usgs-l1-dl") as dc:
             (dataset, error_message) = Doc2Dataset(dc.index)(doc, dst.as_uri())
             dc.index.datasets.update(dataset)
-    else:
-        cmd = ["datacube", "dataset", "update", str(dst), "--location-policy", "forget"]
-        # This avoids update failures due to
-        # minor differences in the extent metadata
-        cmd += INSIGNIFICANT_DIGITS_FIX
-        proc = Popen(cmd, stdout=PIPE, stderr=PIPE)
-        outs, errs = proc.communicate()
-        status = int(proc.returncode)
-        if status > 0:
-            # Move the scene data back to the original location
-            os.rename(dst.parent, src)
-            worked = False
-        update_results = {
-            "cmd": " ".join(cmd),
-            "status": str(status),
-            "outs": str(outs),
-            "errs": str(errs),
-        }
+    """
+
+    cmd = ["datacube", "dataset", "update", str(dst), "--location-policy", "forget"]
+    # This avoids update failures due to
+    # minor differences in the extent metadata
+    cmd += INSIGNIFICANT_DIGITS_FIX
+    proc = Popen(cmd, stdout=PIPE, stderr=PIPE)
+    outs, errs = proc.communicate()
+    status = int(proc.returncode)
+    if status > 0:
```

Comment:
recommendation: should we check for != 0? The reason is that some custom written processes can exit with a negative number (ie. `exit -2`)?

Reply:
thanks, updated